File: QRStockMateApp/app/src/main/java/com/example/qrstockmateapp/api/models/vehicle.kt
package com.example.qrstockmateapp.api.models

data class Vehicle(
    val id: Int,
    val code: String,
    var make: String,  // Fabricante (por ejemplo, Toyota, Ford, etc.)
    var model: String,  // Modelo del vehículo
    var year: Int,  // Año de fabricación
    var color: String,  // Color del vehículo
    var licensePlate: String,  // Matrícula del vehículo
    var maxLoad: Double,  // Carga máxima del vehículo
    val location: String  // Ubicacion cada cierto tiempo
)

File: QRStockMateWeb/src/app/interfaces/vehicle.ts
export interface Vehicle {
    id: number;
    code: string;
    make: string;  // Fabricante (por ejemplo, Toyota, Ford, etc.)
    model: string;  // Modelo del vehículo
    year: number;  // Año de fabricación
    color: string;  // Color del vehículo
    licensePlate: string;  // Matrícula del vehículo
    maxLoad: number;  // Carga máxima del vehículo	
    location: string;  // Ubicacion cada cierto tiempo
}
